teachertool/src/components/DebugInput.tsx

```diff
@@ -22,6 +24,8 @@ const DebugInput: React.FC<IProps> = ({}) => {
 
```

Comment:
Can you move `runEvaluate` into a transform (i.e. to a method defined in `src/transforms/runEvaluate.ts`)? We want to keep application logic out of UI components.